### PR TITLE
:RU: 2 blogs are ready ||Skycoin Launches A Vision For The New Internet|| and ||Skywire: A Free and Open Internet||

### PR DESCRIPTION
**Pull request by rieneuf**

### CODE REVIEW

**Comment by Vittiv** [comment]
В третьей строке не лучше ли написать "недостатки _технологии_ биткойн" и "_приняли участие_ в разработке"?

File: content/statement/Skycoin Launches A Vision For The New Internet.ru.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
++++ title = "Skycoin запускает концепцию нового Интернета " tags = [ "Statement", ] date = "2017-07-24" categories = [ "Statement", ] description = "" +++
```
Comment (gz-c) [inline content/statement/Skycoin Launches A Vision For The New Internet.ru.md]: the formatting of this part is broken. it must be formatted like the other files

**Comment by rieneuf** [comment]
Second blog article added. Previous flaws fixed.

**Comment by gz-c** [comment]
Do you have a telegram username?

**Comment by gz-c** [comment]
5 SKY + 10 SKY = 15 SKY
Please send me your telegram username or give me your sky address here

**Comment by rieneuf** [comment]
2cRLJJ6HdD34esHVs29mkqA3mvcGYFKesBo 
ivan z on Telegram

**Pull request merged**